napari/components/_viewer/qtviewer.py

```diff
@@ -73,6 +78,31 @@ def set_cursor(self, cursor, size=10):
             q_cursor = self._cursors[cursor]
         self.canvas.native.setCursor(q_cursor)
 
+    def screenshot(self, region=None, size=None, bgcolor=None):
+        """Render the scene to an offscreen buffer and return the image array.
```

Comment:
This also seems like it belongs in a separate PR

Reply:
I agree, I'd say for now though we leave it. We will have to make a new PR soon cleaning up the viewer / qtviewer stuff but that refactor can all come then

Reply:
I agree too and why should we leave it? i prefer smaller work on each PR.

Reply:
ok I have moved this back to where it was, we can fix it in the next refactor